Map Version Edits.R

```diff
@@ -173,37 +193,43 @@ dan_func_single_obs <- function(x, parcels_full, parcels_buffered, network) {
       branch = rep(1:4, length.out = n())
     ) %>%
     st_transform(3435)
-  
-  # Find out all the units touched by the cross
+
+  # Step 3: Find all of the neighboring units touched by the cross.
+  # Start by computing the list of parcels that are intersected by the cross
   touching_unit <- suppressWarnings(suppressMessages(st_intersects(cross$geometry, parcels_full)))
-  
-  # Step 3: Find out all the neighbor units for a parcel
+
+  # Compute the neighbor units for a parcel
   neighbor_unit <-
     list(which(replace_na(as.logical(st_intersects(parcels_full, parcels_buffered[x, ])), FALSE)))
   neighbor_unit <- rep(neighbor_unit, each = 4)
-  
-  # Step 4: Find out the intersection of {cross_touching_unit & neighbor_unit}
+
+  # Compute the intersection of {touching_unit & neighbor_unit} to find
+  # neighboring units that intersect the cross
   neighbor_and_touching <- lapply(1:4, function(i) setdiff(intersect(touching_unit[[i]], neighbor_unit[[i]]), x))
-  
-  # Step 5: Remove the cross segments in the intersection of {cross_touching_unit & neighbor_unit}
-  
+
+  # Remove the cross segments from the intersection of
+  # {cross_touching_unit & neighbor_unit}
   cross_int <- replace_na(imap_lgl(neighbor_and_touching, function(y, i) {
     as.logical(st_intersects(cross$geometry[i], parcels_full[i, ]))
   }), FALSE)
-  
+
   cross_int <- map_lgl(neighbor_and_touching, function(x) length(x) > 0)
-  
+
   cross_filter <- cross %>%
     mutate(is_neighboring = cross_int) %>%
     mutate(id = 1:4) %>%
     filter(!is_neighboring)
-  
-  # Step 6: Calculate the angle of cross segments
+
+  # Step 4: Calculate the angle of cross segments in order to filter out
+  # segments that are not at right angles. This is important because we don't
+  # want parallel segments at opposite sides of the parcel, which would not
+  # indicate a corner
```

Comment:
I recall us chatting about how the angle filter may not work properly; is that the case? If so, have we checked in with Dan about that? He didn't seem aware of any such bug when I last discussed the algorithm with him.

Reply:
I talked with @dfsnow about it, and mentioned that I was building in another indicator (which is beginning at line 263).

Reply:
OK cool! I added a quick note on this in 3e25b91506ddbd813c4a84aad8bb84c5471126a6.